webapp-django/crashstats/base/tests/testbase.py

```diff
@@ -21,7 +21,7 @@
                 classes_with_implementation.append(klass)
 
 
-class TestCase(django.utils.unittest.TestCase):
+class TestCase(unittest.TestCase):
 
     def shortDescription(self):
         return None
```

Comment:
We don't use `nose` anymore, so we can drop the `shortDescription`. If you wanted to toss that in this PR, that's cool. Otherwise I'll try to remember to do it later.